DXMainClient/DXGUI/Multiplayer/GameLobby/MultiplayerGameLobby.cs

```diff
@@ -35,9 +35,9 @@ public abstract class MultiplayerGameLobby : GameLobbyBase, ISwitchable
             chatBoxCommands = new List<ChatBoxCommand>
             {
                 new ChatBoxCommand("HIDEMAPS", "Hide map list (game host only)".L10N("UI:Main:ChatboxCommandHideMapsHelp"), true,
-                    s => HideMapList()),
+                    s => ChangeMapListAndOtherControlsVisibility(false)),
                 new ChatBoxCommand("SHOWMAPS", "Show map list (game host only)".L10N("UI:Main:ChatboxCommandShowMapsHelp"), true,
-                    s => ShowMapList()),
+                    s => ChangeMapListAndOtherControlsVisibility(true)),
```

Comment:
No, you're mixing two things. The prefixes you have and the bool you use as a parameter suggest differentiation between host and player, yet you're mixing this with map list hidden or shown.